Xamarin.Forms.Platform.Android/Renderers/ShellFlyoutTemplatedContentRenderer.cs

```diff
@@ -549,8 +549,17 @@ void UpdateElevation()
 
 			protected override void LayoutView(double x, double y, double width, double height)
 			{
+				var context = Context;
+				var paddingLeft = context.FromPixels(PaddingLeft);
```

Comment:
@rmarinho do you remember why you deleted this code here?

https://github.com/xamarin/Xamarin.Forms/commit/2fa7165a99e299ff3a3c20ec3cf10d074cadcdc7#diff-c3567898a88f7ccad57a970d63187eb2d6a8a60fddb83edfb6a4022370f42903L553

It broke the collapse on scroll with Flyout Header

Reply:
This PR also fixes #13561 but adds extraneous paddingLeft to the TitleView - I'm guessing this might potentially have something to do with it

Reply:
If there is a problem that can be demonstrated from that paddingLeft parameter then we can remove but for now all I'm really doing here is restoring the logic that has always been so I'd rather not just remove it for the sake of removing it.

Reply:
@PureWeen  I think it was your commit , https://github.com/xamarin/Xamarin.Forms/pull/13322/commits/cb9aa15bc9e1e5b33195f4cb869da5cd544fe14b

Reply:
@rmarinho  HA

@rachelkang  Are you saying there is a bug with this code?

Reply:
hmm I don't think so actually, nevermind - just realized this is an Android file. Will review the PR more wholly later today!